components/apimgt/org.wso2.carbon.apimgt.rest.api.commons/src/main/java/org/wso2/carbon/apimgt/rest/api/common/APIConstants.java

```diff
@@ -46,7 +46,7 @@
         public static final String TOKEN_SCOPES = "tokenScope";
 
         public static final String TOKEN_1 = "WSO2_AM_TOKEN_1";
-        public static final String TOKEN_2 = "WSO2_AM_TOKEN_2";
+        public static final String TOKEN_2 = "WSO2_AM_TOKEN_MSF4J";
```

Comment:
shall we name this meaningfully something like TOKEN_AM_MSF4J instead of TOKEN_2. Also we have these token names/constants duplicated in the authenticatorConstants right? If so could we make them consistant if we really need to have these duplicated?